style.css

```diff
@@ -159,3 +159,323 @@
   right: 0;
   top: 87.09%;
 }
+
+.work {
+  background-color: rgb(229 229 229);
+  padding: 120px 64px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
+
+.work-title1 {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-bottom: 30px;
+}
+
+.work-head {
+  width: 327px;
+  height: 83px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
```

Comment:
[ ] kindly use the same font defined for the template🤓. 
![headfont](https://user-images.githubusercontent.com/62156607/198576577-c7b636bd-42d2-4af8-b0ca-f7eda59ce8be.PNG)

Reply:
> [ ] kindly use the same font defined for the template🤓. ![headfont](https://user-images.githubusercontent.com/62156607/198576577-c7b636bd-42d2-4af8-b0ca-f7eda59ce8be.PNG)



Is it a must that I use the same font? 
Can't I choose?